js/modal.js

```diff
@@ -0,0 +1,141 @@
+const projectname = document.querySelector('.projectname');
+const workSections = document.querySelector('.project-section');
+const modal = document.querySelector('.modal-example');
+const modalImg = document.querySelector('.modal-img');
+const company = document.querySelector('.company');
+const role = document.querySelector('.role');
+const year = document.querySelector('.year');
+
+const projects = [
+  {
+    name: 'Tonic',
+    imageLink: 'assets/work1.svg',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back end dev',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry',
+    company: 'Canopy',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Multi-post stories',
+    imageLink: 'assets/work2.svg',
+    technologies: ['html', 'css', 'javascript'],
+    role: 'Back End Dev',
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Facebook 360',
+    imageLink: 'assets/work3.svg',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    role: 'Full Stack Dev',
+    description:
+      'Exploring the future of media in Facebook`s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: '#',
+    demoLink: '#',
+  },
+  {
+    name: 'Uber Navigation',
+    imageLink: 'assets/work4.svg',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    role: 'Lead developer',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    company: 'Uber',
+    year: '2018',
+    sourceLink: '#',
+    demoLink: '#',
```

Comment:
- [ ] [Optional] Everything is good, but when I switch to the mobile version then the `li` tag and buttons are not the same in the Figma design please fix this issue in every card. See the image below for reference.
- [ ] Also fix the spacing. see image in Figma design
### Your design
![your](https://user-images.githubusercontent.com/81727988/209308265-dce55e11-6839-4041-909c-89f249a34b7b.png)
### Fimga design
![fimga](https://user-images.githubusercontent.com/81727988/209308313-703dbb44-eef4-4a50-98d1-ff8b3846deec.png)